config/cmake/addGitHooks.cmake

```
@@ -0,0 +1,36 @@
+cmake_minimum_required(VERSION 2.8.7)
+
+function(set_git_hooks_enabled)
+    execute_process(
+            COMMAND ${GIT_EXECUTABLE} config --local core.hooksPath .githooks
+    )
+    message(STATUS "Git hooks enabled")
+endfunction()
+
+function(set_git_hooks_disabled)
+    execute_process(
+            COMMAND ${GIT_EXECUTABLE} config --local --unset core.hooksPath
+    )
+    message(STATUS "Git hooks disabled")
+endfunction()
+
+
+if (NOT DEFINED CLANG_FORMAT)
+    find_program(CLANG_FORMAT "clang-format")
+endif ()
+
+cmake_dependent_advanced_option(HELICS_ENABLE_GIT_HOOKS "Activate git hooks to run clang-format on committed files." ON "CLANG_FORMAT" OFF)
+
+find_package(Git)
+
+if (GIT_FOUND AND (GIT_VERSION_STRING VERSION_GREATER_EQUAL "2.9"))
```

The older CMake versions we support don't have VERSION_GREATER_EQUAL. I think the simplest alternative is:
```suggestion
if (GIT_FOUND AND (NOT GIT_VERSION_STRING VERSION_LESS "2.9"))
```

---

Good catch, thanks, I've made this update.